spec/values/calculation/cos.hrx

```diff
@@ -0,0 +1,247 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `cos()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> deg/input.scss
+a {b: cos(1deg)}
+
+<===> deg/output.css
+a {
+  b: 0.9998476952;
+}
+
+<===>
+================================================================================
+<===> grad/input.scss
+a {b: cos(1grad)}
+
+<===> grad/output.css
+a {
+  b: 0.9998766325;
+}
+
+<===>
+================================================================================
+<===> rad/input.scss
+a {b: cos(1rad)}
+
+<===> rad/output.css
+a {
+  b: 0.5403023059;
+}
+
+<===>
+================================================================================
+<===> turn/input.scss
+a {b: cos(1turn)}
+
+<===> turn/output.css
+a {
+  b: 1;
+}
+
+<===>
+================================================================================
+<===> negative_infinity/input.scss
+@use "sass:math";
+a {b: cos(math.div(-1, 0))}
```

Comment:
```suggestion
a {b: cos(-infinity)}
```

Sass supports [calculation constants](https://developer.mozilla.org/en-US/docs/Web/CSS/calc-constant) in calculations.